pkg/openstack/loadbalancer.go

```diff
@@ -1678,6 +1693,9 @@ func (lbaas *LbaasV2) ensureOctaviaLoadBalancer(ctx context.Context, clusterName
 	// Use more meaningful name for the load balancer but still need to check the legacy name for backward compatibility.
 	lbName := lbaas.GetLoadBalancerName(ctx, clusterName, service)
 	svcConf.lbName = lbName
+	// Get the LBMethod from the annotation. If not set via the annotation, fallback on the OCCM's configuration
+	svcConf.poolLbMethod = getStringFromServiceAnnotation(service, ServiceAnnotationLoadBalancerLbMethod, lbaas.opts.LBMethod)
```

Comment:
can you also adjust your code according to new changes introduced in #2705? i.e. move this string into the `makeSvcConf` method